components/brave_wallet/browser/swap_service_unittest.cc

```diff
@@ -165,8 +167,8 @@ TEST_F(SwapServiceUnitTest, GetPriceQuoteError) {
 }
 
 TEST_F(SwapServiceUnitTest, GetPriceQuoteUnexpectedReturn) {
-  std::string error = "Could not parse response body: Woot";
-  std::string unexpected_return = "Woot";
+  std::string error = "Could not parse response body: ";
+  std::string unexpected_return = "";
```

Comment:
I don't see why you need to change this line.
https://github.com/brave/brave-core/pull/12685/files#diff-d0ea6f6b17e05aaf7bc5102dfe6db4e8974116245c902eb597019780483412e8R249
it would be cleared out on response

Reply:
moved back thanks